cms/publications/blocks.py

```diff
@@ -41,6 +41,22 @@ class DocumentBlock(StructBlock):
     title = RichTextBlock(required=False)
     document = DocumentChooserBlock()
     summary = RichTextBlock(required=False)
+    extension_data = {
```

Comment:
Nitpicky convention, since this is a constant
```suggestion
    EXTENSION_DATA = {
```